site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -237,6 +237,16 @@ They can do the same for Chrome on desktop by going to
 
 We are working on landing a number of changes on the FedCM.
 
+Several updates have already been applied based on feedback. And also it's
+expected to continue evolving until Q4 2023 at least to stabilize. Please see
+Updates for more details.
+
+* **Change Log**: Federated Credential Management API [updates](/docs/privacy-sandbox/fedcm-updates/).
+  * **Cross-origin iframe support**: Call FedCM API from [cross origin iframe](/docs/privacy-sandbox/fedcm-developer-guide/#call-fedcm-from-within-a-cross-origin-iframe)
```

Comment:
These bullets duplicate the ones below. Just appending them something like `(update)` to each items with respective link will be clearer IMO.